recipe/meta.yaml

```diff
@@ -6,16 +6,17 @@ package:
   version: {{ version }}
 
 source:
-  git_url: https://github.com/dgehringer/sqsgenerator.git
-  git_rev: v{{ version }}-rc.1
+  url: https://github.com/dgehringer/sqsgenerator/archive/refs/tags/v{{ version }}-rc.1.tar.gz
+  sha256: c22b6de9c0eb76917abb3f641516339f62369e1676f0e6f9b79dc32b757546d9
 
 
 build:
   number: 0
   entry_points:
     - sqsgen = sqsgenerator.cli:cli
   script: |
-    git submodule update --init
+    rm -rf vcpkg
+    git clone https://github.com/microsoft/vcpkg.git
```

Comment:
There is a specific conda package for this https://anaconda.org/conda-forge/vcpkg

Reply:
Hi many thanks for pointing that out.

 I actually aware of this package. There are two things why
 - From the latest [vcpkg documentation](https://learn.microsoft.com/en-us/vcpkg/get_started/get-started) MS  strongly suggest to use vcpkg as a subdirectory in the root repo folder for CMake based workflows -> this is quite common
 - The package seems to be outdated. The latest version deployed is [2023.04.15](https://anaconda.org/conda-forge/vcpkg/2023.04.15/download/win-64/vcpkg-2023.04.15-h57928b3_2.conda) while the latest version is [2025.06.13](https://github.com/microsoft/vcpkg/tree/2025.06.13)
   - As vcpkg is package manager that means port files (packages) are outdated one would have to check